### PR TITLE
Bug bounty program

### LINKED ISSUE #3118: Clarify bounty program
https://wiki.polkadot.network/docs/kusama-bug-bounty

This is implied, but we should be more explicit that the Kusama Bug Bounty program does _not_ cover programs written on top of, or that use, Kusama - only bugs within Kusama itself and directly related protocol (AnV, XCM, etc.).

Also, we should add a similar page for the Polkadot Bounty program, or simply update this page to clarify that this bounty program applies to both networks.

### PR DESCRIPTION
**Pull request by emresurmeli**
Handles issue #3118 

1. Add bug bounty program page to Pokadot wiki.
2. Include a note to clarify the program and its coverage

### CODE REVIEW

File: docs/general/kusama/kusama-bug-bounty.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ description: An overview of how you can help catch bugs.
 slug: ../../kusama-bug-bounty
 ---
 
+> Note: The bug bounty program does *not* cover bugs on code bases that are external to or, written 
+> on top of Kusama, or that use Kusama. To be eligable for the bug bounty program the bug has to
```
Comment (profradha) [inline docs/general/kusama/kusama-bug-bounty.md]:
```suggestion
> on top of Kusama, or that use Kusama. To be eligible for the bug bounty program the bug has to
```

File: docs/maintain/maintain-bug-bounty.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+id: maintain-bug-bounty
+title: Bug Bounty
+sidebar_label: Bug Bounty
+description: An overview of how you can help catch bugs.
+slug: ../../maintain-bug-bounty
+---
+
+> Note: The bug bounty program does *not* cover bugs on code bases that are external to or, written 
+> on top of Polkadot, or that use Polkadot. To be eligable for the bug bounty program the bug has to
```
Comment (profradha) [inline docs/maintain/maintain-bug-bounty.md]:
```suggestion
> on top of Polkadot, or that use Polkadot. To be eligible for the bug bounty program the bug has to
```

File: docs/general/kusama/kusama-bug-bounty.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ description: An overview of how you can help catch bugs.
...
+> be a part of the Polkadot codebase, this includes protocols that Polkadot uses such as AnV, XCM,
```
Comment (profradha) [inline docs/general/kusama/kusama-bug-bounty.md]:
```suggestion
> be a part of the Polkadot codebase, this includes protocols that Kusama uses such as AnV, XCM, 
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
@laboon Please verify if this PR addresses the issue. Thanks!

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**